css/base.css

```diff
@@ -8,11 +8,13 @@ body {
 }
 
 #photo-template {
+  float: left; 
   background-color: blue;
 }
 
 section {
   display: inline-block;
+  position: inherit;
```

Comment:
Instead of display inline-block, use float:left instead and remove the position tag.

Reply:
Also add a height of 500px